vsts/build_repo.ps1

```diff
@@ -12,4 +12,4 @@ else
     Write-Host "Pull request build detected"
 }
 
-mvn -DRUN_PROVISIONING_TESTS="$Env:runProvisioningTests" -DRUN_DIGITAL_TESTS="$Env:runDigitalTwinTests" -DRUN_IOTHUB_TESTS="$Env:runIotHubTests" -DIS_PULL_REQUEST="$isPullRequestBuild" install -T 2C `-Dmaven.javadoc.skip
```

Comment:
In case you are curious, the backtick characters here work like escape characters when passing in arguments to maven